README.md

```diff
@@ -5,6 +5,39 @@ It is entirely written in [Go](https://golang.org/) and backed by [Redis](https:
 
 Visit the [wiki](../../wiki/) for documentation.
 
+Getting Started
+-------------------------------------------------------------------------------
+
+Make sure that you have a working [Go
+environment](https://golang.org/doc/install) and that you have configured your
+[`$GOPATH`](https://golang.org/doc/code.html#Workspaces) correctly.
+
+Clone the repository or `go get` it with:
+```shell
+$ go get github.com/skroutz/downloader
+```
+
+Install `librdkafka-dev` in order to be able to build the project. For Debian
```

Comment:
kafka is not a dependency in the `master` branch :) _yet_
Let's add it later...